src/server/worker/client.go

```diff
@@ -0,0 +1,117 @@
+package worker
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/pachyderm/pachyderm/src/client"
+	"github.com/pachyderm/pachyderm/src/client/debug"
+	"github.com/pachyderm/pachyderm/src/client/pps"
+
+	etcd "github.com/coreos/etcd/clientv3"
+	"google.golang.org/grpc"
+)
+
+const (
+	workerEtcdPrefix = "workers"
```

Comment:
Not DRY ... this is repeated here:

https://github.com/pachyderm/pachyderm/blob/bd3b6c5bb57e98a9d327c360ab555ac98a3f232f/src/server/cmd/worker/main.go#L203

Reply:
Good catch, will fix.